src/storage/backend/redis/Redis.h

```diff
@@ -41,6 +41,7 @@ class Redis final : public Backend, public iosource::IOSource {
     void HandleGetResult(redisReply* reply, ResultCallback* callback);
     void HandleEraseResult(redisReply* reply, ResultCallback* callback);
     void HandleGeneric(redisReply* reply);
+    void HandleInfoResult(redisReply* reply);
```

Comment:
I know that this is probably a lost fight in this cause - but I want to point out that, in principle, it would be neat for functions in header files to contain comments, following our [style guide](https://github.com/zeek/zeek/wiki/Coding-Style-and-Conventions#commenting).

Same applies below.